code_samples/user_management/oauth_google/config/custom_services.yaml

```
@@ -10,4 +10,4 @@ services:
         arguments:
             $template: '@@ibexadesign/account/login/oauth2_login.html.twig'
         tags:
-            - { name: ibexa.admin_ui.component, group: login-form-after }
+            - { name: ibexa.twig.component, group: login-form-after }
```

`login-form-after` is BC for `admin-ui-login-form-after`?
```suggestion
            - { name: ibexa.twig.component, group: login-form-after }
```

---

I'll update it to use up-to-date values - but the BC layer is here: https://github.com/ibexa/admin-ui/blob/bb2de218e1f6ba8d2d5d4e9ebeabfc6b21e5a956/src/lib/Component/Registry.php